ETC.tsv

```diff
@@ -107,7 +107,7 @@ xETC_20150317_000106	Kepas have aged the Pokusaku flower through constantly leac
 ETC_20150317_000107	Gelsvas Old Kepa
 ETC_20150317_000108	Black Old Kepa
 ETC_20150317_000109	Red Old Kepa
-xETC_20150317_000110	Its dilapidated color from aging through life only substantiate that it is dangerous.
+xETC_20150317_000110	Its dilapidated color from aging through life only substantiate its precariousness.
```

Comment:
substantiate - > substantiates 
precariousness is another mouthful, staying with dangerous is fine for now.